test_prebuild/test_blocked_data/CMakeLists.txt

```diff
@@ -1,14 +1,21 @@
 #------------------------------------------------------------------------------
-cmake_minimum_required(VERSION 3.0)
+cmake_minimum_required(VERSION 3.10)
 
 project(ccpp_blocked_data
         VERSION 1.0.0
-        LANGUAGES Fortran)
+        LANGUAGES C Fortran)
```

Comment:
Note. I will clean this and line 14 up in a follow-up PR; similar for `test_chunked_data`.